test/replication.test.js

```diff
@@ -75,7 +75,8 @@ describe('Replication / Change APIs', function() {
         if (err) return done(err);
         SourceModel.changes(FUTURE_CHECKPOINT, {}, function(err, changes) {
           if (err) return done(err);
-          expect(changes).to.be.empty();
+          /*jshint -W030 */
+          expect(changes).to.be.empty;
```

Comment:
This is weird, why did you have to replace the function call `empty()` with a property access `empty`? Are you sure this test is not passing just because `empty` is not defined at all?

The same comment applies to other similar changes in the test files.